src/main/java/com/microsoft/sqlserver/jdbc/Geography.java

```diff
@@ -118,14 +135,15 @@ public static Geography parse(String wkt) throws SQLServerException {
     }
 
     /**
-     * Constructs a Geography instance that represents a Point instance from its X and Y values and an SRID.
+     * Constructor for a Geography instance that represents a Point instance from its latitude and longitude values and
+     * a Spatial Reference Identifier.
      * 
      * @param x
-     *        x coordinate
+     *        latitude
      * @param y
-     *        y coordinate
+     *        longitude
      * @param srid
-     *        SRID
+     *        Spatial Reference Identifier value
      * @return Geography Geography instance
      * @throws SQLServerException
      *         if an exception occurs
```

Comment:
I guess you missed out on the method params: x and y should be changed to latitude/longitude respectively.
Refer: [MS Docs](https://docs.microsoft.com/en-us/sql/t-sql/spatial-geography/point-geography-data-type?view=sql-server-2017)

Reply:
done